### PR TITLE
Alias e2e tests

### PR DESCRIPTION
**Pull request by dolanor**
Add e2e tests for Alias.

The biggest change on the service side was to create an empty slice for the ListAliases so the JSON returns `[]` and not `null`.

### CODE REVIEW

File: tests/e2e/alias_test.go
Comment (ggarri) [inline tests/e2e/alias_test.go:70]: Please use the e2e standards we use in other e2e test units https://github.com/ConsenSys/quorum-key-manager/blob/main/tests/e2e/ethereum_test.go#L112
```
- should {action} successfully
- should failed [with error type] {when|if} {failure scenario} 
```

**Review by ggarri — APPROVED** [review]
Please make sure you update the e2e test unit message to align with the rest of test units

**Comment by dolanor** [comment]
Updated test message.
Waiting for end of CI before merging.

**Pull request merged**
